pyqt-configure

```diff
@@ -2,7 +2,6 @@
 
 ${PYTHON} configure.py \
     --assume-shared \
-    --concatenate \
```

Comment:
Your commit message does not explain why you're removing this option, please do.